### ISSUE TITLE
Python driver `next(wait=1)` not working for changefeeds

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#3936]
The following never returns for me in the Python driver:
```python
import rethinkdb as r
r.connect().repl()
r.table_create('bob')
alpha = r.table('bob').changes().run()
alpha.next(wait=1)
```

**Comment by Tryneus** [comment]
The fix for this is up in review 2720, along with a fix for #3812, which was a problem for the newly-added tests.

**Comment by Tryneus** [comment]
This has been fixed in `next` in commit 1e3d822, and cherry-picked into `v2.0.x` in commit 56d55fd (tests in commits ea4a8da and eccacbd, respectively).  Will be in release 2.0.
